lib/Widgets/Settings.vala

```diff
@@ -96,9 +115,47 @@ namespace Granite {
             }
         }
 
+        private void setup_accent_color () {
+            try {
+                if (portal == null) {
+                    portal = Portal.Settings.get ();
+                }
+
+                var variant = portal.read (
+                    "org.freedesktop.appearance",
+                    "accent-color"
+                ).get_variant ();
+
+                accent_color = parse_color (variant);
+
+                portal.setting_changed.connect ((scheme, key, value) => {
+                    if (scheme == "org.freedesktop.appearance" && key == "accent-color") {
+                        accent_color = parse_color (value);
+                    }
+                });
+            } catch (Error e) {
+                warning (e.message);
+
+                // Set a default in case we can't get from system
+                accent_color = Gdk.RGBA ();
+                accent_color.parse ("#3689e6");
```

Comment:
Sorry, I tested this fallback by throwing a new error in L128 and found this fallback does not work as expected; this sets anything to accent_color, which results it to be "rgb(0, 0, 0)".

I confirmed the following code works as expected instead:

```suggestion
                _accent_color = Gdk.RGBA ();
                _accent_color.parse ("#3689e6");
```